### PR TITLE
Report correct next update times

### PR DESCRIPTION
**Pull request by timbru**

### CODE REVIEW

File: src/daemon/ca/publishing.rs [hunk 1]
```diff
@@ -21,23 +21,10 @@ use rpki::repository::{
...
-            ResourceClassName, Revocation, Revocations,
-        },
-        crypto::KrillSigner,
-        error::Error,
-        eventsourcing::{KeyStoreKey, KeyValueStore, PreSaveEventListener},
-        KrillResult,
-    },
-    constants::CA_OBJECTS_DIR,
-    daemon::{
+use crate::{commons::{KrillResult, api::{Base64, Handle, IssuedCert, ObjectName, RcvdCert, RepositoryContact, ResourceClassName, Revocation, Revocations, Timestamp, rrdp::PublishElement}, crypto::KrillSigner, error::Error, eventsourcing::{KeyStoreKey, KeyValueStore, PreSaveEventListener}}, constants::CA_OBJECTS_DIR, daemon::{
```
Comment (ximon18) [inline src/daemon/ca/publishing.rs:24]: This doesn't look like it's been formatted.

Comment (timbru) [reply src/daemon/ca/publishing.rs:24]: ah, yes, I had to disable format on save whilst working on rpki-rs.. I will push a fix later

File: src/daemon/ca/publishing.rs [hunk 2]
```diff
@@ -357,6 +344,24 @@ impl CaObjects {
         all_elements
     }
 
+    /// Returns the closest next update time from among manifests held by this CA
+    pub fn closest_next_update(&self) -> Option<Timestamp> {
+        let mut closest = None;
+
+        for rco in self.classes.values() {
```
Comment (ximon18) [inline src/daemon/ca/publishing.rs:351]: What is an "rco" ? "resource class something" ?

**Review by ximon18 — APPROVED** [review]

File: src/daemon/ca/publishing.rs (hunk 2 above)
Comment (timbru) [reply src/daemon/ca/publishing.rs:351]: I could use `resource_class_objects` it's a bit wordy but probably better

Comment (timbru) [reply src/daemon/ca/publishing.rs]: done

File: src/daemon/ca/publishing.rs (hunk 1 above)
Comment (timbru) [reply src/daemon/ca/publishing.rs]: cargo fmt done
